src/components/players/detailsPlaceholder.module.scss

```diff
@@ -0,0 +1,58 @@
+@import 'src/scss/variables.scss';
+
+.playerDetailsContainer {
+    min-width: 400px;
+    padding: 10px;
+}
+
+.picture {
+    height: 292px;
+    background-color: $grey1b;
+}
+
+.underline {
+    height: 3px;
+    margin-top: 5px;
+    background-color: $grey1b;
+}
+
+.box {
+    display: flex;
+    margin-top: 22px;
+}
+
+.textBox {
+    display: flex;
+    flex-direction: column;
+    margin: 0 2px;
+}
+
+.text1 {
+    width: 202px;
+    height: 30px;
+    background-color: $grey1b;
+}
+
+.text2 {
+    width: 112px;
+    height: 18px;
+    margin-top: 5px;
+    background-color: $grey1b;
+}
+
+.text3 {
+    width: 88px;
+    height: 12px;
+    margin-top: 5px;
+    background-color: $grey1b;
+}
+
+.circle {
+    height: 58px;
+    width: 58px;
+    flex: 1;
+    border-radius: 50%;
+    margin-left: calc(51% - 58px);
```

Comment:
Don't use `margin-left` like this. As a matter of fact the circle looks more like an oval because of the css here. Instead use flexbox to align all the items and add additional divs if necessary

Reply:
Okay, this works but no need for extra div. It should be like this in html:

```
            <div className={styles.box}>
                <div className={styles.textBox}>
                    <div className={styles.barOne} />
                    <div className={styles.barTwo} />
                    <div className={styles.barThree} />
                </div>
                <div className={styles.circle} />
            </div>
```

and then in scss for styles.box

```
.box {
    display: flex;
    justify-content: space-between;
    margin-top: 20px;
}
```